src/services/mail/MailService.ts

```diff
@@ -0,0 +1,7 @@
+export interface MailService {
+  sendMail(
+    senderEmail: string,
+    recipientEmail: string,
+    messageType: string,
+  ): Promise<any>;
```

Comment:
This is nitpicking but if we don't expect a result, set it to return Promise<void>.